docsrc/imap/reference/manpages/configs/cyrus.conf.rst

```diff
@@ -219,6 +219,11 @@ scheduled cleanup/maintenance.
         The command (with options) to spawn as a child process.  This
         string argument is required.
 
+        Note that this argument is parsed quite naively, only trivial commands
+        are likely to work.  If you need to run anything complex here, make a
+        shell script that calls the real command, and set the event to call
+        the script.
```

Comment:
It might be nice if this was more explicit about what won't work, ie "Don't use spaces, or quote characters, or..."

Reply:
I feel like leaving it ambiguous and "here be dragons"-ey means I'm not accidentally implying that anything I didn't list works.  I haven't actually thought much about the set of what-won't-work beyond observing that it's not the empty set, and the issue isn't new (but does become more likely to be tripped over as people stick more arbitrary non-Cyrus stuff into cyrus.conf EVENTS).

Reply:
My problem with this warning is it tells a user "This may surprise you!" but does not at all provide any sort of guidance or expectations. I think "Well, I'll just use a shell script. But wait? Is it so naive I can't put a _ or a - in the name of my script? I guess I have to experiment to find out... how frustrating".

Reply:
I've rephrased this paragraph to hopefully clarify exactly what's naive about it.